src/components/NewCommentForm.tsx

```diff
@@ -72,28 +166,43 @@
 
         <div className="control">
           <textarea
+            value={commentText}
+            onChange={handleCommentTextChange}
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames('textarea', {
+              'is-danger': commentTextError.length > 0,
+            })}
           />
         </div>
-
-        <p className="help is-danger" data-cy="ErrorMessage">
-          Enter some text
-        </p>
+        {commentTextError.length > 0 && (
+          <p className="help is-danger" data-cy="ErrorMessage">
+            Enter some text
+          </p>
```

Comment:
Instead of hardcoding the error message 'Enter some text', consider dynamically displaying the `commentTextError` message. This will make the error handling more flexible and informative.